src/java/org/apache/cassandra/db/compaction/CompactionHistoryTabularData.java

```diff
@@ -74,10 +74,11 @@ public static TabularData from(UntypedResultSet resultSet) throws OpenDataExcept
             long bytesIn = row.getLong(ITEM_NAMES[4]);
             long bytesOut = row.getLong(ITEM_NAMES[5]);
             Map<Integer, Long> rowMerged = row.getMap(ITEM_NAMES[6], Int32Type.instance, LongType.instance);
-
+            Map<String, String> compactionProperties = row.getMap(ITEM_NAMES[7], UTF8Type.instance, UTF8Type.instance);
             result.put(new CompositeDataSupport(COMPOSITE_TYPE, ITEM_NAMES,
                        new Object[]{ id.toString(), ksName, cfName, compactedAt, bytesIn, bytesOut,
-                                     "{" + FBUtilities.toString(rowMerged) + "}" }));
+                                     '{' + FBUtilities.toString(rowMerged) + '}' ,
```

Comment:
formatting, it should be

````
'{' + FBUtilities.toString(rowMerged) + '}',
'{' + FBUtilities.toString(compactionProperties) + '}' }));
````

1) there is redundant space at `'}' ,`
2) the second line is not aligned wit the first one
````